packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/components/NameAndUiLinkDetailsItem.tsx

```diff
@@ -12,18 +13,18 @@ import { ProviderDetailsItemProps } from './ProviderDetailsItem';
 
 /**
  * @typedef {Object} NameDetailsItemProps - extends ProviderDetailsItemProps
-
- * @property {string} [webUILinkText - A text to be displayed as a content.
- * @property {function} [calcWebUILink] - A method for auto calculating the provider's UI path.
- **/
+ *
+ * @property {string} [webUILinkText - A label text to be displayed as a content.
+ * @property {function} [calcWebUILink] - A method for auto calculating the provider's UI path (used as a default if no value is set by the user).
+ */
 
 export interface NameAndUiLinkDetailsItemProps extends ProviderDetailsItemProps {
   webUILinkText?: string;
   calcWebUILink?: (provider: V1beta1Provider) => string;
 }
 
 const getProviderUIAnnotation = (provider: V1beta1Provider): string =>
-  provider?.metadata?.annotations?.['forklift.konveyor.io/providerUI'];
+  provider?.metadata?.annotations?.[PROVIDER_UI_ANNOTATIONS];
```

Comment:
```suggestion
  provider?.metadata?.annotations?.['forklift.konveyor.io/providerUI'];
```

no need for a const here, it's just a string, my concern here is that if we change PROVIDER_UI_ANNOTATIONS we will not see it here and we may use the wrong string

Reply:
The motivation for using const instead of a hard coded string is that in case of changing the annotation, it can be changed in one place instead of three and it is less buggy. 
If someone will change the const without checking where it's used all around the code then it can be done with the string as well.....and it's a coding problem anyway.
I like using consts but sure I can switch back to a string.